src/udisksdaemon.c

```diff
@@ -310,6 +310,36 @@ check_modules_state_in_idle_cb (gpointer user_data)
   return G_SOURCE_REMOVE;
 }
 
+static void
+bd_log_redirect (gint level, const gchar *msg)
+{
+  switch (level)
+    {
+      case BD_UTILS_LOG_EMERG:
+      case BD_UTILS_LOG_ALERT:
+      case BD_UTILS_LOG_CRIT:
+        udisks_critical ("[blockdev] %s", msg);
+        break;
+      case BD_UTILS_LOG_ERR:
+        udisks_error ("[blockdev] %s", msg);
```

Comment:
Note that this will abort the process, is that desired? `udisks_critical()` is actually lower severity.

Reply:
Definitely not desired, I forgot `udisks_error` works this way. I changed everything above warning to `udisks_critical`.